raiden/utils/nursery.py

```diff
@@ -146,6 +148,19 @@ def _free_resources(self) -> None:
             for p in self._processes:
                 p.send_signal(signal.SIGINT)
 
-            for p in self._processes:
-                if p.wait() != STATUS_CODE_FOR_SUCCESS:
-                    print("Process did not exit cleanly", p.communicate())
+            try:
+                for p in self._processes:
+                    exit_code = p.wait(timeout=20)
```

Comment:
In the SP we had to increase the stop timeout to 10min because raiden can take a long time to shut down if a transfer is waiting for the lock to expire.

Reply:
I can make that configurable, so that the Janitor can be used in all such cases.

Do we really need to wait that long in the SP? If we are trying to stop nodes while they are waiting for a lock expiry, something else must have gone wrong, already. Can't we just kill the nodes in that case?

Reply:
Hummm, IMO the timeouts should be set as high in the stack as possible. Timeouts simply don't compose well and passing the configuration every where is a burden.